package.json

```diff
@@ -40,7 +40,8 @@
     "mustache": "^2.2.1",
     "mutationobserver-shim": "^0.3.1",
     "pbf": "^1.3.2",
-    "point-in-polygon": "1.0.0",
+    "point-geometry": "^0.0.0",
+    "point-in-polygon": "steve9164/point-in-polygon",
```

Comment:
From the [`vector-tile` readme] (https://github.com/mapbox/vector-tile-js/blob/master/README.md)
#### Methods

- **loadGeometry()** &mdash; parses feature geometry and returns an array of
  [Point](https://github.com/mapbox/point-geometry) arrays (with each point having `x` and `y` properties)